bin/install-chart.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+CHART="$1"
+shift # preserve "$@" for --post-renderer-args
+
+GENESTACK_DIR="${GENESTACK_DIR:-/opt/genestack}"
+CHART_META_FILE=\
+"${CHART_META_FILE:-$GENESTACK_DIR/bin/chart-install-meta.yaml}"
+GENESTACK_DIR="${GENESTACK_DIR:-/opt/genestack}"
+GENESTACK_CONFIG_DIR="${GENESTACK_CONFIG_DIR:-/etc/genestack}"
+GENESTACK_CHART_DIR=\
+"${GENESTACK_CHART_DIR:-$GENESTACK_DIR/base-helm-configs/$CHART}"
+GENESTACK_CHART_CONFIG_DIR=\
+"${GENESTACK_CHART_CONFIG_DIR:-$GENESTACK_CONFIG_DIR/helm-configs/$CHART}"
+# This Python needs PyYAML, which the normal Genestack venv will have.
+YAML_PARSER_PYTHON="${YAML_PARSER_PYTHON:-$(which python)}"
+YAML_PARSER_PY="${YAML_PARSER_PY:-$GENESTACK_DIR/bin/yamlparse.py}"
+YAML_PARSER_CMD="${YAML_PARSER_CMD:-$YAML_PARSER_PYTHON $YAML_PARSER_PY}"
+
+IFS= readarray -t chart_values < <($YAML_PARSER_CMD "$CHART_META_FILE" "$CHART")
+
+values_length=${#chart_values[@]}
+
+# This for loop takes values from the YAML file `namespace: prometheus` and
+# sets variables like NAMESPACE=prometheus for use by the script.
+for (( i=0; i<values_length; i+=2 ))
+do
+    var_name="$(echo "${chart_values[i]}" | tr '[:lower:]' '[:upper:]')"
+    declare "$var_name=${chart_values[i+1]}"
+done
+
```

Comment:
We can add something like the following to ensure that required variables for helm intall command, exist after parsing the YAML
```suggestion
required_vars=("NAMESPACE" "REPONAME" "RELEASENAME" "URL")

for var in "${required_vars[@]}"; do
    if [[ -z "${!var}" ]]; then
        echo "Error: Required variable $var is missing in the YAML file."
        exit 1
    fi
done
```

Reply:
added